source/editor/dataproviders/list_all_influxdbs.php

```diff
@@ -98,40 +98,15 @@ function query( $q, $db = '', $auth )
   $resSeries = array();
   $measurements = array();
 
-  $seriesArr = json_decode( query( 'SHOW SERIES', $database, $_GET['auth'] ), true );
+  $seriesArr = json_decode( query( 'SHOW MEASUREMENTS', $database, $_GET['auth'] ), true );
   $series = $seriesArr[ 'results' ][ 0 ][ 'series' ][ 0 ][ 'values' ];
   if( NULL != $series )
   {
     foreach( $series as $thisSeries )
     {
-      $list = explode( ',', $thisSeries[ 0 ] );
-      $measurement = array_shift( $list );
-      if( !array_key_exists( $measurement, $measurements ) )
-        $measurements[ $measurement ] = array();
-
-      foreach( $list as $tag )
-      {
-        $tagKV = explode( '=', $tag );
-        if( array_key_exists( $tagKV[ 0 ], $measurements[ $measurement ] ) )
-        {
-          $measurements[ $measurement ][ $tagKV[ 0 ] ][ $tagKV[ 1 ] ] = 1; // fake set operation
-        } else
-        {
-          $measurements[ $measurement ][ $tagKV[ 0 ] ] = array( $tagKV[ 1 ] => 1 ); // fake set operation
-        }
-      }
-    }
-    // translate fake set to real set/array
-    foreach( $measurements as $measurement => $measurementValues )
-    {
-      foreach( $measurementValues as $tag => $tagValues )
-        $measurements[ $measurement ][ $tag ] = array_keys( $tagValues );
-      $resSeries[ $measurement ] = $measurements[ $measurement ];
-
-      // now forget all the nice information and compact to the relevant one:
       $arrData[] = array(
-        'value' => $database . '/' . $measurement,
-        'label' => $database . '/' . $measurement
+        'value' => $database . '/' . $thisSeries[ 0 ],
+        'label' => $database . '/' . $thisSeries[ 0 ]
       );
     }
   }
```

Comment:
This Dataprovider is deprecated und will not be used by the new manager. Please apply those changes to `source/rest/manager/src/Api/DataproviderApi.php`.

Reply:
I applied the same change to `source/rest/manager/src/Api/DataproviderApi.php` now - but I couldn't test it there as my code uses `source/editor/dataproviders/list_all_influxdbs.php`.

Reply:
I can't test it either because I do not have an setup with influxdb. but you can just replace the URL with `source/rest/manager/index.php/data/influxdbs` and must see the same result. It should be tested, before we merge it because it seems to be broken anyways. I just wanted to open the URL with the current develop-version and  get an error: `Call to undefined function OpenAPIServer\Api\query()` maybe you can have a look at it.

Reply:
The PHP namespace seems to do strange things (probably it's working fine but as I'm not used to them it looks rather strange to me)

A fix is commited. I don't like the necessary switch from `require_once(hidden.php)` to `require`. But I had no way to make `$hidden` visible inside of the `query` function...